doc/man3/SSL_CONF_cmd.pod

```diff
@@ -85,6 +85,15 @@ its preference list. This usually indicates a client without AES hardware
 acceleration (e.g. mobile) is in use. Equivalent to B<SSL_OP_PRIORITIZE_CHACHA>.
 Only used by servers. Requires B<-serverpref>.
 
+=item B<-ignore_unexpected_eof>
+
+Some TLS implementations do not send close_notify on shutdown. Before OpenSSL
+3.0, SSL_ERROR_SYSCALL with errno=0 was reported in such cases. This
+behaviour contradicts the B<SSL_ERROR_SYSCALL> documentation. Since OpenSSL 3.0
+the returned error is B<SSL_ERROR_SSL> with a meaningful error on the error
+stack. This option gives the legacy error in such cases. Equivalent to
```

Comment:
`This option gives the legacy error in such cases.` This is incorrect - SSL_ERROR_ZERO_RETURN should be returned with this option. Also please add some warning that using this options is safe only when the underlying protocol detects truncation by itself.

Reply:
Fixed. Thanks!

Reply:
Fixed. Thanks!